rotkehlchen/globaldb/handler.py

```diff
@@ -165,9 +165,9 @@ def add_asset(
             data: Union[EvmToken, Dict[str, Any]],
     ) -> None:
         """
-        Add an asset in the DB. Either an ethereum token or a custom asset.
+        Add an asset in the DB. Either an ethereum token or a user asset.
```

Comment:
so for what assets is this function used? Crypto/ethereum/evm tokens .... basically the one that the frontend allows users to input today? There is also an elif, for custom assets, so should it not also be explained in the docstring? Generally this docstring is confusing me a bit

Reply:
the `elif` is for a different method( the `retrieve_assets()`)